x-pack/legacy/plugins/siem/public/components/page/network/domains_table/index.test.tsx

```diff
@@ -25,6 +25,12 @@ import { mockDomainsData } from './mock';
 
 jest.mock('../../../../lib/settings/use_kibana_ui_setting');
 
+jest.mock('../../../search_bar', () => ({
+  siemFilterManager: {
+    addFilters: jest.fn(),
+  },
+}));
+
```

Comment:
Domains table is gonezo as part of https://github.com/elastic/kibana/pull/47608, which is now merged. so a pull from `master` should clean this up.